log/handler_glog.go

```diff
@@ -39,9 +39,9 @@ var errTraceSyntax = errors.New("expect file.go:234")
 type GlogHandler struct {
 	origin Handler // The origin handler this wraps
 
-	level     uint32 // Current log level, atomically accessible
-	override  uint32 // Flag whether overrides are used, atomically accessible
-	backtrace uint32 // Flag whether backtrace location is set
+	level     atomic.Uint32 // Current log level, atomically accessible
+	override  atomic.Uint32 // Flag whether overrides are used, atomically accessible
```

Comment:
This could also be a boolean. Instead of setting it to `len(filter)`, we would set it to `len(filter) != 0`